.gitlab/ci/test.yml

```diff
@@ -135,21 +76,19 @@ test_run:
     - . /usr/workspace/benchpark-dev/benchpark-venv/$SYS_TYPE/bin/activate
     # Initialize System
     - |
+        EXTRA_ARGS=""
         if [ "$HOST" == "tioga" ]; then
-            ./bin/benchpark system init --dest=tioga-system ${ARCHCONFIG} ~gtl
-            system_id=$(./bin/benchpark system id ./tioga-system)
-        else
-            ./bin/benchpark system init --dest=$HOST-system ${ARCHCONFIG} cluster=$HOST
-            system_id=$(./bin/benchpark system id ./$HOST-system)
+            EXTRA_ARGS="~gtl"
         fi
+        ./bin/benchpark system init --dest=$HOST-system ${ARCHCONFIG} cluster=$HOST $EXTRA_ARGS
     # Initialize Experiment
     - ./bin/benchpark experiment init --dest=${BENCHMARK}-benchmark ${BENCHMARK}+${VARIANT}
     # Build Workspace
     - ./bin/benchpark setup ${BENCHMARK}-benchmark ${HOST}-system workspace/
     # Setup Ramble & Spack
     - . workspace/setup.sh
     # Setup Workspace
-    - cd ./workspace/${BENCHMARK}-benchmark/${system_id}/workspace/
+    - cd ./workspace/${BENCHMARK}-benchmark/$HOST-system/workspace/
```

Comment:
add curly brackets here for consistency

Reply:
[3317313](https://github.com/LLNL/benchpark/pull/669/commits/3317313d02df3a305eb2cf2a039659f4ba3f3b0a)